networking/ipv6.md

```diff
@@ -1,145 +1,134 @@
+   | `FELIX_IPV6SUPPORT` | `true` |
+
+1. Apply the edited manifest with `kubectl apply -f`.  
+   New pods will get IPv6 addresses as well as IPv4, and can communicate with each other and the outside world over IPv6.
+
+#### Enable IPv6-only, initial install, Kubernetes
+
+To configure IPv6-only support for Kubernetes during {{site.prodname}} installation, follow these steps.
+
+1. Download the appropriate {{site.prodname}} manifest for IPv6 deployment and save it as `calico.yaml`.
+1. Edit the `calico.yaml` ipam section, and modify it to [disable IPv4 assignments and enable IPv6 assignments](/reference/cni-plugin/configuration#ipam).
+   ```
+       "ipam": {
+           "type": "calico-ipam",
+           "assign_ipv4": "true",
+           "assign_ipv6": "true"
+       },
+   ```
+1. Add the following environment variables to the calico-node Daemonset in the `calico.yaml` file.   
+Be sure to set the value for `CALICO_IPV6POOL_CIDR` to the desired IP pool; it should match the `--cluster-cidr` passed to the kube-controller-manager and to kube-proxy.
 
    ```yaml
    - name: CALICO_IPV6POOL_CIDR
      value: "fd20::0/112"
    ```
-
-1. Ensure in the `calico.yaml` file that the environment variable
-   `FELIX_IPV6SUPPORT` is set `true` on the calico-node DaemonSet.
+1. In the `calico.yaml` file, verify that the environment variable `FELIX_IPV6SUPPORT` is set `true` on the calico-node Daemonset.
 1. Apply the `calico.yaml` manifest with `kubectl apply -f calico.yaml`.
 
->**Note**: For clusters provisioned with kubeadm, {{site.prodname}} autodetects the IPv4 and IPv6 pod CIDRs and does not require configuration.
-{: .alert .alert-info}
-
-#### Using only IPv6
+#### Enable IPv6-only, after install, Kubernetes
 
-If you wish to only use IPv6 (by disabling IPv4) or your hosts only have
-IPv6 addresses, you must disable autodetection of IPv4 by setting `IP`
-to `none`.
+If you installed {{site.prodname}} on the cluster using the default IPv4, and you want switch to IPv6-only, or your hosts only have IPv6 addresses, follow these steps.
 
-With IPv4 enabled, Calico uses the node's IPv4 address as the BGP router ID. With IPv4 disabled,
-you must configure another method to calculate the BGP router ID. There are two ways to do this:
+1. Disable [IP autodetection of IPv4]({{site.baseurl}}//networking/ip-autodetection) by setting `IP` to `none`.
+1. Calculate the {{site.prodname}} BGP router ID for IPv6 using either of the following methods.
+   - Set the environment variable `CALICO_ROUTER_ID=hash` on {{site.nodecontainer}}.   
+  This configures {{site.prodname}} to calculate the router ID based on the hostname, or 
+   - Pass a unique value for `CALICO_ROUTER_ID` to each node individually. 
+            |  
+4. If you are using [kube-dns](/getting-started/kubernetes/installation/manifests/kubedns.yaml), you may need to modify your DNS for IPv6 operation.
```

Comment:
As previously indicated, I think this kube-dns material should be deleted.  (Has anyone else said that it should stay?)

Reply:
This is all "old" content that has not been reviewed in ages. It was the last doc in Alex's "reorg work" last year. I merely moved content around, so that explains why so much FUD in the content.